frontend/src/app/main/ui/workspace/tokens/modals/themes.cljs

```diff
@@ -208,7 +210,11 @@
   (let [{:keys [dropdown-open? _on-open-dropdown on-close-dropdown on-toggle-dropdown]} (wtco/use-dropdown-open-state)
         theme (ctob/make-token-theme :name "")
         on-back #(set-state (constantly {:type :themes-overview}))
-        on-submit #(st/emit! (wdt/create-token-theme %))
+        on-submit (mf/use-fn
+                   (fn [theme]
+                     (st/emit! (ptk/event ::ev/event {::ev/name "create-theme-tokens"}))
```

Comment:
I think the name should be `create-tokens-theme`

Reply:
These are the names defined in the story, I'll ask @myfunnyandy to reconfirm